packages/lit-dev-content/site/docs/tools/server-side-rendering.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Server-side Rendering
+eleventyNavigation:
+  key: Server-side rendering
+  parent: Tools
+  order: 9
+---
+
+Server-side rendering (SSR) Lit templates and elements is possible with the [`@lit-labs/ssr`](/docs/libraries/labs/) package. As with all `@lit-labs` packages, it is considered experimental and not ready for production use but we encourage you to try it out and file any issues for bugs or feature requests.
+
+## SSR Methods
+The following methods of rendering are provided by `@lit-labs/ssr` for rendering Lit elements and templates on the server.
```

Comment:
I think before this there should be some discussion of the DOM shim and differences between global and worker scopes wrt the shim. This whole section could be a new page too, with the preceding being a short overview.